demos/supabase-edge-function-auth/README.md

```diff
@@ -0,0 +1,21 @@
+# PowerSync + Supabase Anonymous Auth
+
+Demo app demonstrating use of JWT authentication using Supabase wth the PowerSync SDK for Flutter.
+
+# Setup
+
+* Create an edge auth function using <https://github.com/powersync-ja/powersync-jwks-example>
+
+# Running the app
+
+Install the Flutter SDK, then:
+
+```sh
+cp lib/app_config_template.dart lib/app_config.dart
+flutter pub get
+flutter run
+```
+
+# Configure the app
+
+Copy the contents of  [lib/app_config_template.dart](./lib/app_config_template.dart) into `lib/app_config.dart`, and alter the values to use the credentials of your new Supabase project.
```

Comment:
duplicate step, this was already done on line 14

Reply:
Will fix in the rename PR